packages/pico8-ls/package.yaml

```
@@ -0,0 +1,19 @@
+---
+name: pico8-ls
+description: Full language support for the PICO-8 dialect of Lua.
+homepage: https://github.com/japhib/pico8-ls
+licenses:
+  - MIT
+languages:
+  - PICO-8
+  - PICO-8 Lua
+categories:
+  - LSP
+source:
+  # renovate:datasource=git-refs
+  id: pkg:github/japhib/pico8-ls@ef01bb52cba6fe358dcc2478ace74033744fdd26
+  build:
+    run: |
+      npm run compile
+bin:
+  pico8_ls: node:client/out-min/main.js
```

This will be bothersome to change in the future. Seeing as they don't provide an executable yet (so we don't know what name it would potentially be), I think it'd be better to use the more conventional `-` separator.

```suggestion
  pico8-ls: node:client/out-min/main.js
```